rskj-core/src/test/java/org/ethereum/rpc/JsonRpcNettyServerTest.java

```diff
@@ -52,7 +62,33 @@ public void smokeTestUsingInvalidContentType() throws Exception {
         smokeTest("text/plain");
     }
 
-    public void smokeTest(String contentType) throws Exception {
+    @Test
+    public void smokeTestUsingValidHost() throws Exception {
+        smokeTest(APPLICATION_JSON, "localhost");
+    }
+
+    @Test(expected = IOException.class)
+    public void smokeTestUsingInvalidHost() throws Exception {
+        smokeTest(APPLICATION_JSON, "evil.com");
+    }
+
+    @Test
+    public void smokeTestUsingValidHostAndHostName() throws Exception {
+        smokeTest(APPLICATION_JSON, "www.google.com", InetAddress.getByName("www.google.com"), new ArrayList<>());
```

Comment:
This test depends on the network, right? I think we shouldn't do any IO in tests.